### PR TITLE
Update logging samples to fit new style guide and match Node.js samples.

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

**Comment by jonparrott** [comment]
Sorry for the big change, I screwed myself because I wrote half of these before I realized there were existing samples.

**Comment by waprin** [comment]
This is going to break the include tags on https://cloud.google.com/logging/docs/api/tasks/creating-logs

**Comment by jonparrott** [comment]
I know, I'll handle updating docs.

File: logging/cloud-client/snippets.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+
+def list_entries(logger_name):
+    """Lists the most recent entries for a given logger."""
+    logging_client = logging.Client()
+    logger = logging_client.logger(logger_name)
+
+    print('Latest entries for logger {}:'.format(logger.name))
+
+    entries, token = logging_client.list_entries(
```
Comment (waprin) [inline logging/cloud-client/snippets.py]: Two things:

1) If you don't loop on the token, you're going to have a bad time. The API will return 0 entries at first then return 5 on the next one, the token is also used to indicate "loading"

2) You should just be able to do logger.list() now. There was a gcloud-python bug I fixed that should be released now)

https://github.com/GoogleCloudPlatform/gcloud-python/pull/1848

Comment (jonparrott) [inline logging/cloud-client/snippets.py]: ugh.

Comment (jonparrott) [inline logging/cloud-client/snippets.py]: Fixed.

**Comment by jonparrott** [comment]
@waprin updated, PTAL.

File: logging/cloud-client/export_test.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+def test_update(example_sink, capsys):
+    updated_filter = 'severity>=INFO'
+    export.update_sink(TEST_SINK_NAME, updated_filter)
+
+    example_sink.reload()
+    assert example_sink.filter_ == updated_filter
+
+
+def ttest_delete(example_sink, capsys):
```
Comment (waprin) [inline logging/cloud-client/export_test.py]: typo in test name

File: storage/api/README.md [hunk 3]
```diff
@@ -10,3 +16,31 @@ These samples are used on the following documentation pages:
...
 
 <!-- end-auto-doc-link -->
+
+## Running the samples
+
+1. Your environment must be setup with [authentication
+information](https://developers.google.com/identity/protocols/application-default-credentials#howtheywork). If you're running in your local development environment and you have the [Google Cloud SDK](https://cloud.google.com/sdk/) installed, you can do this easily by running:
+
+        $ gcloud init
+
```
Comment (waprin) [inline storage/api/README.md]: is this our standard now? What I dislike about it is it only really makes sense the first time, if you already have a configuration, it's going to create a new one every time.

Comment (jonparrott) [inline storage/api/README.md]: No, I just copypastad this. I'll update it to use `application-default`.

File: storage/api/README.md [hunk 4]
```diff
@@ -1,4 +1,10 @@
-# Cloud Storage API Samples
+# Stackdriver Logging v1 Samples
```
Comment (waprin) [inline storage/api/README.md]: I think you messed something up, this is the storage README

File: logging/api-client/README.md [hunk 5]
```diff
@@ -1,17 +1,28 @@
-# Google Cloud Logging v1 Samples
+# Stackdriver Logging v1 Samples
 
 **Note that these samples are for the v1 Samples, using the Google API Client. 
  It's recommended you instead use the [Logging v2 samples](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/logging/api), which use the Google 
  Cloud client library.**
 
 This section contains samples for [Google Cloud Logging](https://cloud.google.com/logging).
```
Comment (pfritzsche) [inline logging/api-client/README.md]: s/Google Cloud/Stackdriver/

?

**Comment by waprin** [comment]
code lgtm, offline organization comments

**Pull request merged**